src/media/css/marketplace-elements.styl

```diff
@@ -0,0 +1,384 @@
+
+    .mkt-prompt-content {
+        background: $white;
+        padding-bottom: 10px;
+
+        > form:first-child,
+        > div:first-child {
+            padding: 5px 20px;
+        }
+    }
+    form > div:not(.mkt-prompt-btn-wrap) {
+        margin-bottom: 20px;
+        width: 100%;
+    }
+    h3 {
+        font-weight: $font-bold;
+        margin-top: 20px;
+    }
+    p {
+        font-size: $font-size;
+        line-height: 22px;
+        margin: 15px 0;
+    }
+    textarea {
+        border: 1px solid $greyscale-light-grey;
+        -moz-box-sizing: border-box;
+        -webkit-box-sizing: border-box;
+        box-sizing: border-box;
+        font-family: $primary-font-family;
+        margin-bottom: 25px;
+        min-height: 180px;
+        padding: 10px;
+        width: 100%;
+    }
+    select {
+        width: 100%;
+    }
+}
+
+.mkt-prompt-btn-wrap {
+    display: inline-flex;
+    width: 100%;
+}
+
+.mkt-prompt-btn-wrap button {
+    border: 0;
+    border-radius: 5px;
+    color: $white;
+    flex: 1;
+    font-size: 13px;
+    font-weight: $font-bold;
+    height: 48px;
+
+    &:hover {
+        cursor: pointer;
+    }
+    &[data-type="cancel"] {
+        // Cancel button.
+        display: none;  // Don't show cancel button when as a page.
+        background: $action-positive;
+
+        &:hover {
+            background: $action-positive-hover;
+        }
+        &:active {
+            background: $action-positive-tapped;
+        }
+    }
+    &[type="submit"] {
+        background: $action-success;
+
+        &:hover {
+            background: $action-success-hover;
+        }
+        &:active {
+            background: $action-success-tapped;
+        }
+    }
+}
+
+.mkt-prompt form:invalid [type="submit"] {
+    background: $greyscale-grey;
+
+    &:hover {
+        cursor: default;
+    }
+}
+
+.mkt-prompt[data-modal] {
+    background: rgba($greyscale-grey, .8);
+    height: 100%;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1000;  // TODO: write a z-index system.
+
+    .mkt-prompt-content {
+        border-radius: 0;
+        padding-bottom: 0;
+        position: absolute;
+        left: 0;
+        margin: 5% auto 0;
+        max-width: ($max-site-width / 2)px;
+        right: 0;
+        width: 90%;
+    }
+    .mkt-prompt-btn-wrap {
+        margin-bottom: -5px;
+        position: relative;
+        right: 20px;
+        width: unquote('calc(100% + 40px)');
+
+        +rtl() {
+            left: 20px;
+            right: inherit;
+        }
+    }
+    [data-type="cancel"] {
+        display: inline-block;
+    }
+}
+
+@media $base-tablet {
+    .mkt-prompt .mkt-prompt-content {
+        margin-bottom: 20px;
+    }
+    .mkt-prompt[data-modal] .mkt-prompt-content {
+        width: (2 / 3 * 100)%;
+    }
+    .mkt-prompt-btn-wrap {
+        justify-content: center;
+
+        button[type="submit"] {
+            max-width: $mobile-content-width;
+        }
+    }
+}
```

Comment:
Are you sure you want this file to be added? Looks like these changes are not related to this bug.